examples/label_studio_annotation/README.md

```diff
@@ -177,55 +161,6 @@ S3_BUCKET_NAME=<YOUR_DESIRED_S3_BUCKET_NAME>
 aws s3api create-bucket --bucket=$S3_BUCKET_NAME --region=$REGION --create-bucket-configuration=LocationConstraint=$REGION
 ```
 
-Create a `.json` file on your local hard drive containing the following data,
```

Comment:
In this new approach, Label Studio uses the AWS access key that you explicitly configure for the S3 artifact store, so you no longer need to generate a session token especially for Label Studio. I figured all these instructions were part of generating the AWS session token and no longer needed, but please let me know if that's not the case.

Reply:
I'm thinking that CORS will not be enabled by default on the S3 artifact store via our usual credentials, as explained [here](https://labelstud.io/guide/storage.html#Configure-access-to-your-S3-bucket). (step 3).

Reply:
I don't think I understand why this is needed. I didn't need to setup CORS for any of my other buckets and I'm accessing them from all over the place - my local host, external Kubernetes clusters in AWS, GCP and Azure - without problems.

Reply:
Done. I wonder what would break in Label Studio if we don't add this. I don't think it was set up for the bucket I was using and didn't see anything obvious breaking.